react/features/base/participants/functions.ts

```diff
@@ -497,6 +497,37 @@ export function getSourceNamesByMediaType(
         .map(s => s[0]);
 }
 
+/**
+ * Returns a list of source names associated with a given remote participant and for the given video type (only for
+ * video sources).
+ *
+ * @param {(Function|Object)} stateful - The (whole) redux state, or redux's {@code getState} function to be used to
+ * retrieve the state.
+ * @param {string} id - The id of the participant whose source names are to be retrieved.
+ * @param {string} videoType - The type of video, camera or desktop.
+ * @returns {Array<string>|undefined}
```

Comment:
Wouldn't it be better for the consumer of this function if we return an empty array here?